test/service.thrift

```diff
@@ -0,0 +1,7 @@
+
+service Foo {
+    byte foo(0: byte number) throws (
```

Comment:
And the generated code uses `-1` for the field ID for some reason:

    oprot.writeFieldBegin('foo', TType.STRING, -1)